recipes/management/increasing_the_capacity_to_win/growing/expectations/onboarding/README.md

```diff
@@ -0,0 +1,34 @@
+
+# Recipes
+
+## Dobromir's Recipes
+When I was hired by Wade Chambers at TellApart I was shocked to receive, on the first day, this well-thought out 'Onboarding Document'. How cool was that! It had very cleary expectations for the first few days and the first month. I've basically copied their approach in my recipe, with a few minor changes. See the examples below.
+
+1. Start with something inspiring. Explain why they were hired. Don't be afraid to call out the Big Problem or Big Challenge: people want to know that they matter! 
+1. Give them Day 1 objectives. What do you want them to accomplish on their first day? This can really set the tone of your culture. E.g. at TellApart they encouraged everyone to commit a change to the repo on Day
+    1. You should at least have all the 'orientation' stuff set up on Day 1, i.e. where the code is, where the goals are, etc.
+1. Give them Week 1 objectives. 
+    1. This should be something small, that a ramped engineer can do in 1-2 days. A good candidate is that P2 bug that has been open forever, or to do a release of a particularly complex piece of software.
+1. [Optional] give them 2nd & 3rd week objectives. 
+    1. For more junior employees, it helps to break down where they can contribute week-by-week. For more senior employees, I might skip the mid-month objectives.
+1. Set 30 day goals & check-in
+    1. These should be as concrete as possible. For more junior engineers it might be as simple as implementing a mid-sized feature. For senior people it might be publishing a Technical Design Doc. For every senior employees, I've let them define their own 30-day goals. See example below.
+    1. Put a 30-day check-in on calendar when they start, so they know that the goals will be reviewed.
+1. [OPTIONAL] set 90 day objectives & check-in.
+    1. This can be helpful if you don't have an OKR process or other way of keeping people accountable after the first month is over.
+1. Share the doc with the _entire_ team. 
+    1. This is another great way to both keep people looped-in about what new employees are doing, and help peer pressure encourage folks to hit their goals.
+1. Review the doc every week. 
+    1. I ask people to cross out what they've done and let me know where they are stuck.
+    1. The doc can become stale quickly -- that's OK. Your best employees will figure out what needs to be done better than you could have defined it for them. Give them the freedom to change the plan -- but still document the plan, so you don't run into the risk of them __not__ acomplishing anything in the first 30 days. 
+1. Request their own 30-day retro.
+    1. I have all new employees share their thoughts about the team/company before they've completely 'gone native'. See my recipe under [Retro Culture](need_link). Example below.
+
```

Comment:
The list is completely missing the people side for me - given we work in tech and people will naturally bias towards code submitted over people met it may be good to explicitely call out that building relationships and understanding of the immediate and braoder team is a tangible goal too. WDYT?

Reply:
I think that's a good idea -- you should write your own recipe about it :)